webodf/lib/gui/TrivialUndoManager.js

```diff
@@ -195,12 +312,38 @@ gui.TrivialUndoManager = function TrivialUndoManager(defaultRules) {
         eventNotifier.unsubscribe(signal, callback);
     };
 
+
+    /**
+     * @return {!boolean}
+     */
+    this.isDocumentModified = isModified;
+
+    /**
+     * @param {!boolean} modified
+     * @return {undefined}
+     */
+    this.setDocumentModified = function(modified) {
```

Comment:
Yes, I have at least one usecase:
when loading a new document that has been created from some non-static parameters, e.g. user input, one should be able to set the initial version already as modified, so that any other customer logic can continue to rely on the flag of the editor, instead to having to care for (and maintain) another flag.
That specific use-case could of course alternatively be solved by having an optional parameter on the loadDoc call, but passing that on to the undomanager instance also needs care and in the end will need similar if-then-else code somewhere in the undo manager.
Thinking further, another use case might be that while the document is edited, it might get set a new reference (for persistence or further processing), against which is not yet synced/flushed. Not backed by real use case, but might be possible.

So in total this opens more flexibility. The only danger I see is that it might result in programming errors due to inverted bool flags, e.g. `editor.setDocumentModified(!notSaved)`, which could result in data loss. Not sure programmers really need protection against that, at least cannot remember any API with similar logic I used had special handling.

So for now would (well, given you :shipit: will :) ) just stay with this API. Let's see how real world feedback will shape it further.